extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQControllerTask.java

```diff
@@ -147,6 +149,24 @@ public String getType()
     return TYPE;
   }
 
+  @Override
+  public String getLabel() {
```

Comment:
Is laning available only for MSQ tasks for now?

Reply:
It applies to all task types. For non-MSQ task label is same as task type (can refer to the default method added in `Task` interface)